benchmarks/dynamo/ci_expected_accuracy/aot_eager_torchbench_inference.csv

```diff
@@ -82,7 +82,7 @@ detectron2_fasterrcnn_r_50_fpn,eager_fail_to_run,0
 
 
 
-detectron2_fcos_r_50_fpn,pass,21
+detectron2_fcos_r_50_fpn,pass,22
```

Comment:
This is fine. We are tracing more code, so new graph breaks. I will fix them in later PRs.